### ISSUE TITLE
Iowa Copyright Review

### ISSUE THREAD

**Issue by justinelliotmeyers** [openaddresses/openaddresses#4988]
We are missing tons of resources in Iowa. Looks like a ftp went down that provided us numerous counties. I found a state wide repository, but the terms seem like we cannot use the data. Can someone confirm that we cannot use this?

https://www.iowagisdata.org/gisdr/userterms

Thanks,
Justin

**Comment by iandees** [comment]
Yep, looks like they say no one can use the data in any way without permission. We could ask for permission to share the data with OpenAddresses, but it's not there by default.

**Comment by justinelliotmeyers** [comment]
@iandees thanks! Will email and ask.

**Comment by justinelliotmeyers** [comment]
had a phone call with :
Micah Van Maanen
Information Technology Director
Sioux County, Iowa
P: 712-737-6818 | C: 712-360-1090

super nice guy. So the data cannot be compiled and reserved holistically. In individual counties in the 'OPEN' section, where terms allow it, it can be! He told me this over the phone, so I am not sure if we would need it in writing.

I did ask him about some of the e911 data their state has. He said thats a great place to start as well. So I'll keep going that route for now. Then I might explore individual counties here with 'OPEN' status and where terms look okay.

I will start with Polk County, as their gis person told me to grab it from here this morning.
